kitchen.yml

```diff
@@ -15,7 +15,24 @@ platforms:
   - name: fedora-29
   - name: opensuse-leap-42
   - name: ubuntu-16.04
+    driver:
```

Comment:
you don't need URLs for the bento boxes. kitchen-vagrant defaults to bento if you don't provide a URL

Reply:
I might have had some issue with my local install and needed these to make the tests work.

Reply:
6e275f7b77cb09b85fe507b024253e67b73e23ee removed the URls